backend/migrations/env.py

```
@@ -0,0 +1,113 @@
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+logger = logging.getLogger('alembic.env')
+
+
+def get_engine():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_engine`

(ANN201)